src/Node/Stream/StdIO.purs

```diff
@@ -0,0 +1,11 @@
+-- | The standard IO streams for the current process.
+module Node.Stream.StdIO where
```

Comment:
:+1: Ok, done. Agree re gzip and crypto. I think they would be pretty straightforward to wrap too, which is nice.